Documentation/faq.rst

```diff
@@ -0,0 +1,35 @@
+.. todo::
+  Create new subsections as needed.
+
+===
+FAQ
+===
+
+NSH Tricks
+==========
+
+  How to enable interrupt an NSH Application using Ctrl^C ?
```

Comment:
Each question should be in separate subsection. Use `-` underline.
Also, for key indications and menuconfig entries use the roles mentioned here: https://nuttx.apache.org/docs/latest/contributing/documentation.html#user-indications